index.html

```diff
@@ -1381,12 +1381,9 @@ <h3>Algorithm</h3>
                   an <a>IRI</a>
                   or <a>blank node identifier</a>, the <a>vocabulary mapping</a>
                   of <var>result</var> is set to
-                  <span class="changed">the result of using the 
-                    <a href="#iri-expansion">IRI Expansion algorithm</a>,
-                    passing <var>result</var> for <var>active context</var>,
-                    <var>value</var>,
-                    <code>true</code> for <var>vocab</var>, and
-                    <code>true</code> for <var>document relative</var>.
+                  <span class="changed">the result of
+                    <a>iri expanding</a> <var>value</var>
+                    using <code>true</code> for <var>document relative</var>.
```

Comment:
Double period. I think this is the one to remove.
```suggestion
                    using <code>true</code> for <var>document relative</var>
```